src/chi/components/brand/brand.scss

```diff
@@ -0,0 +1,21 @@
+@import 'mixins';
+@import 'variables';
+
+$brand-sizes: (xs: 0.75rem, sm: 1rem, sm2: 1.25rem, sm3: 1.5rem, md: 2rem, lg: 4rem, xl: 6rem, xxl: 8rem);
+
+.chi {
+  .a-brand {
+    & > img {
+      height: 2rem;
+      width: auto;
+    }
+
```

Comment:
Let's add a `-fluid` modifier class to the img with a value of `max-width: 100%;` for responsive support on larger sizes. @jllr Should we make this an image utility and store in utilities?

Reply:
@mattnickles it makes sense to store the -fluid modifier class in utilities, so we have it isolated from a-brand and can work with other images. Apart from add max-width: 100%; it should be added a height: auto; so the img resize with the container properly.